source/parsing/ParserBase.cpp

```diff
@@ -109,7 +109,7 @@ Token ParserBase::expect(TokenKind kind) {
             // become unbalanced and flush it.
             openDelims.clear();
         }
-        lastPoppedDelims = {};
```

Comment:
I have no idea why this is needed but this line causes an error under Clang. (I don't have the exact error recorded but it was something about an ambiguous overload.)